src/docs/extract.ml

```diff
@@ -133,8 +136,19 @@ struct
         {
           it = Syntax.LetD ({ it = Syntax.VarP { it = name; _ }; _ }, rhs, _);
           _;
-        } ->
-        Some (mk_xref (Xref.XValue name), extract_let_doc rhs name)
+        } -> (
+        match rhs with
+        | Source.{ it = Syntax.ObjBlockE (sort, fields); _ } ->
```

Comment:
This is the sort field you want to use. You can assert on sort Type.Memory, which is a hacky label only used to describe stable variable records internally.